src/modules/Thread/components/ParentMessageInfo/index.tsx

```diff
@@ -192,7 +208,11 @@ export default function ParentMessageInfo({
   }
 
   return (
-    <div className={`sendbird-parent-message-info ${className}`}>
+    <div
+      className={`sendbird-parent-message-info ${className}`}
+      {...(isMobile) ? { ...longPress } : {}}
```

Comment:
```suggestion
      {...isMobile ? longPress : {}}
```
I guess you can get the same result \w the above :)